Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -609,13 +609,17 @@ class UnrealResourcePreparer
     }
   }
 
-  virtual void*
-  prepareRasterInLoadThread(const CesiumGltf::ImageCesium& image) override {
+  virtual void* prepareRasterInLoadThread(
+      const CesiumGltf::ImageCesium& image,
+      void* pRendererOptions) override {
+    auto options =
+        reinterpret_cast<FRasterOverlayRendererOptions*>(pRendererOptions);
```

Comment:
For me the problem was that when prepareRasterInLoadThread is called, I'm not sure which raster overlay it is referring to. Each raster overlay can have different texture properties, but if we store the properties in the UnrealResourcePreparer, then it would become a global option.